### DISCUSSION TITLE
Dragging links to new tab in compact mode

### DISCUSSION THREAD

**Discussion by SobhanBose** [zen-browser/desktop#2303] [Ideas]
Dragging links onto the new tab button works just fine unless you are in compact mode. When in compact mode, the toolbar never expands when draging a link and hence we cannot drop it onto the new tab button.

**Comment by eandersons** [comment]
I would like to add that similar issue (also described in #2524) is when trying to move tabs between windows with compact mode enabled.

So a solution could be to slide out the tab bar whenever a link or tab is dragged over a Zen window when compact mode is enabled.
